pkg/controller/webhook/reconcile_test.go

```diff
@@ -128,3 +130,119 @@ func verifyCertificates(t *testing.T, rootCert []byte, serverCert []byte) {
 	_, err = cert.Verify(opts)
 	assert.NoError(t, err)
 }
+
+func TestUpdateOperatorPod(t *testing.T) {
```

Comment:
Have you considered testing `TestUpdateOperatorPods` instead to get a bit more test coverage?

Reply:
Yes, I updated the test.